qt/applications/workbench/workbench/plugins/workspacewidget.py

```diff
@@ -383,7 +383,13 @@ def _action_double_click_workspace(self, name):
                     plot_kwargs = {"axis": MantidAxType.BIN}
                     plot([ws], errors=False, overplot=False, wksp_indices=[0], plot_kwargs=plot_kwargs)
             else:
-                plot_from_names([name], errors=False, overplot=False, show_colorfill_btn=True)
+                try:
+                    plot_from_names([name], errors=False, overplot=False, show_colorfill_btn=True)
+                except RuntimeError as err:
+                    if "Variable invalidated" in str(err):
+                        logger.error(str(err))
+                    else:
+                        raise err
```

Comment:
Here I opted to only cover the `plot_fom_names` function inside the `try` block. 
- Should the `try` block be bigger?
Then I have an if/else statement to check that the `RunTime` error caught is specifically the one relating to the deleted data. 
- Should I use a regular expression in this if statement instead?
- If in the future the message for this runtime error changes, then the if statement will always yield false, is there any way to prevent this?